dmutils/email.py

```diff
@@ -57,12 +57,12 @@ def generate_token(data, secret_key, salt):
     return ts.dumps(data, salt=salt)
 
 
-def decode_token(token, secret_key, salt):
+def decode_token(token, secret_key, salt, max_age=86400):
```

Comment:
Could you name this `max_age_in_seconds`, if the unit is seconds?

Or call the function with a sensible unit, eg `decode_token(max_age_in_hours=48)` and calculate the seconds inside the function?